### ISSUE TITLE
Filtered format: AND/OR switch does not show

### ISSUE THREAD

**Issue by RomStrom** [SemanticMediaWiki/SemanticResultFormats#264]
### Setup
- MW version: 1.27.2 (1c409c5) 20:42, 6 April 2017
- DB (MySQL etc.): 5.5.52-MariaDB
- PHP version: 5.6.30 (apache2handler)
- SMW version: 2.5.2 (84007ca)02:45, 17 May 2017
- SRF version: 2.4.2 (17be353) 10:13, 6 May 2017
- Browsers and versions (if applicable): Chrome, Version 59.0.3071.115 (Official Build) (32-bit)

**The Issue**
When adding the filter option to have an AND / OR switch visible for a certain property, the expected AND / OR radio buttons do not appear, as they do [here](http://aroundisleofwight.info/en/Days_Out) under "Facilities".

**Steps To Reproduce (in my version)**
The manner in which I am calling this option is as follows:

```
{{#ask: [[Sample Property::+]]
|?Property1 |+filter=value |+value filter switches=and or
|?Property2 |+filter=value
|format=filtered
|views=list
|list view type=list
}}
```

We should expect Property1 to have and AND / OR switch just above the selecting list of values wiki-wide assigned to Property1. However, only this selection list appears and not the switch,

**Steps To Reproduce in Sandbox**
The issue does not reproduce in the sandbox, nor does the filtered SRF function overall as this [test query](https://sandbox.semantic-mediawiki.org/wiki/Test1221_Query) shows.

This query uses the following code.

```
{{#ask: [[Category:Test1221]]
|?Has text |+filter=value, +value filter switches=and or
|format=filtered
|views=list
|list view type=list
}}
```

**Expected result:**
AND/OR radio buttons appear and allow switching between AND or OR logic within single-property filtering, as shown [here](http://aroundisleofwight.info/en/Days_Out) under "Facilities".

**Observed result:**
In my version, radio buttons do not appear. In SMW sandbox, filtered format does not work properly as shown [here](https://sandbox.semantic-mediawiki.org/wiki/Test1221_Query).

**Comment by kghbln** [comment]
@RomStrom Thank you for reporting. The filtered format was overhauled in the meantime meaning the version served with 2.5.2 is different and the one on 2.4.2 will no longer be touched. Please try to verify if the issue is still present with SRF 2.5.2. Thanks a lot!

@s7eph4n FYI

**Comment by kghbln** [comment]
@RomStrom Ah, ok. So it is not working either on SRF 2.5.2

**Comment by s7eph4n** [comment]
@kghbln: The Filtered format was turned off by default in 2.5 as it accesses an external map service. It needs to be [turned on explicitly](https://www.semantic-mediawiki.org/wiki/Extension:Semantic_Result_Formats#Enabling_additional_formats). Could you do that in the sandbox?

**Comment by s7eph4n** [comment]
Probably connected to #189.

**Comment by kghbln** [comment]
@kghbln: The Filtered format was turned off by default in 2.5 as it accesses an external map service. It needs to be turned on explicitly. Could you do that in the sandbox?

I thought that I had already done it: [Now active!](https://sandbox.semantic-mediawiki.org/w/index.php?title=SMW_Sandbox%3ASetup_information&action=historysubmit&type=revision&diff=6987&oldid=6896)

**Comment by RomStrom** [comment]
Testing my [modified test page](https://sandbox.semantic-mediawiki.org/wiki/Test1221_Query) now, it seems the default logic between a single property's values is back to OR. However, the AND/OR switch seems to appear but not quite work correctly.

**Comment by s7eph4n** [comment]
Fixed for both master and 2.5.x
